src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_tasks_callbacks.c

```diff
@@ -119,11 +119,14 @@ cJSON* wm_agent_upgrade_update_status_success_callback(int *error, cJSON* input_
         char *buffer = NULL;
 
         os_calloc(OS_MAXSTR, sizeof(char), buffer);
-        snprintf(buffer, OS_MAXSTR, "%03d com clear_upgrade_result -1", agent_id);
-
+        cJSON *command_info = cJSON_CreateObject();
```

Comment:
I suggest to create a function that parses the message to be sent to the agent. Take a look at `wm_agent_upgrade_parse_task_module_request`.